packages/models/schemas/entity.json

```
@@ -0,0 +1,54 @@
+{
+  "$id": "entity",
+  "definitions": {
+    "refTuple": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "id",
+        "type"
+      ]
+    },
+    "refString": {
+      "type": "string"
+    },
+    "reference": {
+      "oneOf": [
+        { "$ref": "#/definitions/refTuple" },
+        { "$ref": "#/definitions/refString" }
+      ]
+    },
+    "relations": {
+      "type": "object",
+      "additionalProperties": {
+        "oneOf": [
+          { "$ref": "#/definitions/reference" },
+          {
+            "type": "array",
+            "items": { "$ref": "#/definitions/reference" }
+          }
+        ]
+      }
+    },
+    "attributes": {
+      "type": "object"
+    }
+  },
+  "type": "object",
+  "properties": {
+    "ref": { "$ref": "#/definitions/reference" },
```

Why shorthand notation here, while not `attrs` for attributes or `rel` for relations?

```suggestion
    "reference": { "$ref": "#/definitions/reference" },
```